addons/hr_holidays/static/src/views/kanban/hr_holidays_kanban_controller.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="hr_holidays.HrHolidaysKanbanController.Buttons" t-inherit="web.KanbanView.Buttons" t-inherit-mode="primary" owl="1">
+        <xpath expr="//button[hasclass('o-kanban-button-new')]" position="replace">
+            <button type="button" class="btn btn-primary" t-on-click="() => this.onClickAddEmployeeToPlan(props.context)">
```

Comment:
Isn't it possible to use a python method through the `name` attribute instead of using js? It feels overkill.

Reply:
New string fix inside Kanban controller and list controller.  [link](https://github.com/odoo/odoo/blob/master/addons/web/static/src/views/kanban/kanban_controller.xml#L10) We need an 'Employee to Plan' instead of a New. Second think that this is a template so the name attribute won't work directly. As

`<button type="button" name="action_abc" class="btn btn-primary">  Add Employee To plan</button>`

I  take reference from website slides [ref](https://github.com/odoo/odoo/blob/master/addons/website_slides/static/src/views/slide_channel_partner_list/slide_channel_partner_list_view.xml#L6-L10)

Let me know if I am doing something wrong.